index.php

```diff
@@ -18,7 +18,10 @@
 </head>
 <body>
     <nav>
-        <h1>Icepace</h1>
+        <a class="homePageTitle" href="index.php">
+            <h1>Icepace</h1>
+        </a>
+        <a class="signUp" href="registrationPage.php">sign up</a>
```

Comment:
this button is currently only on the homepage - it needs to be on every page

Reply:
fixed.